### PR TITLE
feat(forms): Various improvements

### LINKED ISSUE #1250: forms: create copy
If a form has ended, show that at the bottom with some text, and with a button to copy over the questions to a new form

### LINKED ISSUE #1267: Make it possible to add a branding to a section
pollbranding

- [x] rogerthat-backend (API & CAPI)
- [x] rogerthat-ios-client
- [ ] rogerthat-android-client
  - add branding (2px shadow underneath the logo and avatar in all native brandings)
  - center the button in the first and submission sections
  - use "Start" as caption of the 1st section button
    - TBD: caption customizable via API and OCA sets "Start" as first section button caption #1270?
- [ ] oca-backend (dashboard)
  - let people upload a picture when creating a section (ratio 640x240)
  - recolor the icon with the app's tint color

poll-icon

### LINKED ISSUE #1268: Make a form's news item sticky until the end date (cities only)
Only for a poll set by a city with an end time

### LINKED ISSUE #1272: Use poll icon as section avatar
See #1267

### LINKED ISSUE #1273: Implement 5 steps when creating a form
Like discussed before

1. Maak uw enquete/poll/inschrijvingsformulier aan
2. Test bij u of iemand anders
3. Indien test klaar, stel optioneel de einddatum in
4. Wenst u nog een actie te verbinden aan uw formulier?
5. Maak uw enquete/poll/inschrijvingsformulier kenbaar door het uitsturen van uw nieuwsbericht.

### LINKED ISSUE #1277: Show image button also when a form isn't saved yet

### LINKED ISSUE #1280: forms: implement 'Next action' user interface
For single select questions, you should be able to add a 'next action'
On a section, you should be able to set the next action as well.

Next action: submit the form, go to a specific section, or just continue to the next section

### LINKED ISSUE #1281: forms: enable step 5
disable the switch to place it in the app, with a tooltip stating that they have to test first

### LINKED ISSUE #1286: forms: Individual form responses
UI for viewing individual form responses
- 1 per "page"
- option to delete a response (delete on rogerthat too)
- option to delete all responses

### PR DESCRIPTION
**Pull request by lucasvanhalst**
Closes #1250
Closes #1267
Closes #1268
Closes #1272
Closes #1273
Closes #1277
Closes #1280
Closes #1281
Closes #1286

Needs https://github.com/our-city-app/mobicage-backend/pull/180

Needs migration:
```python
def rebuild_stats():
    from solutions.common.migrations._003_rebuild_form_stats import rebuild_form_statistics
    return rebuild_form_statistics(False)
```

### CODE REVIEW

**Comment by lucasvanhalst** [comment]
Available to test on https://forms-dot-rogerthat-server.appspot.com

File: src/solutions/common/localizer.py [hunk 1]
```diff
@@ -4163,7 +4076,7 @@
          u'Info sessions': u'Infosessies',
          u'Input': u'Invoer',
          u'Internal account': u'Interne rekening',
-         u'Invalid broadcast type': u'Ongeldige categorie voor dit bericht. Indien u net een nieuw notificatietype heeft toegevoegd, gelieve dan eerst te publiceren.',
+         u'Invalid broadcast type': u'Ongeldige categorie voor dit nieuwsbericht. Indien u net een nieuw notificatietype heeft toegevoegd, gelieve dan eerst te publiceren.',
```
Comment (bart-at-mobicage) [inline src/solutions/common/localizer.py]: This reverts #1283. Looks like you forgot to update the translations when doing #1283

File: src/solutions/common/localizer.py [hunk 2]
```diff
@@ -5029,6 +4945,124 @@
...
+         u'oca.please_select_an_email_address': u'Selecteer een e-mailadres uit de lijst',
+         u'oca.preview_form': u'Bekijk op mobiel',
+         u'oca.previously_upload_images': u'Eerder ge\xfcploade afbeeldingen',
+         u'oca.question': u'Vraag',
+         u'oca.questions': u'Vragen',
+         u'oca.readonly_for_tombola': u'Deze instelling kan niet aangepast worden wanneer de tombola aan staat.',
+         u'oca.reorder_sections': u'Herorder pagina\'s',
+         u'oca.required': u'Verplicht',
+         u'oca.response_count.none': u'Nog geen antwoorden',
+         u'oca.response_validation': u'Antwoord instellingen',
```
Comment (bart-at-mobicage) [inline src/solutions/common/localizer.py]: should be 1 word

File: src/solutions/common/localizer.py [hunk 3]
```diff
@@ -5515,7 +5548,7 @@
          u'you_have_been_granted_permission_to_service': u'U heeft nu toegang tot het dashboard van %(service)s.',
          u'your-menu': u'jouw menu',
          u'your_application_is_ready': u'Uw eigen app is een feit!',
-         u'your_registration_request_submitted': u'Beste, proficiat met uw applicatie. Van zodra uw stad of gemeente uw aanvraag heeft goedgekeurd zal u een email ontvangen met uw login en paswoord, zo kan u van start. Nadien zullen wij u per mail informeren over alle mogelijkheden binnen uw eigen applicatie. Mogelijkheden zoals een gratis e-shop, een gratis reservatiesysteem, hoe u berichten kan uitsturen... Tot binnenkort. Veel succes en welkom!',
+         u'your_registration_request_submitted': u'Beste, proficiat met uw applicatie. Van zodra uw stad of gemeente uw aanvraag heeft goedgekeurd zal u een email ontvangen met uw login en paswoord, zo kan u van start. Nadien zullen wij u per mail informeren over alle mogelijkheden binnen uw eigen applicatie. Mogelijkheden zoals een gratis e-shop, hoe u nieuws en acties kan uitsturen, een gratis klantenkaartsysteem... Tot binnenkort. Veel succes en welkom!',
```
Comment (bart-at-mobicage) [inline src/solutions/common/localizer.py]: when/why did this string change? has it been changed in mobitrans, but not in the code or the other way around?

Comment (lucasvanhalst) [reply src/solutions/common/localizer.py]: probably the first

**Comment by lucasvanhalst** [comment]
Closing this in favour of #1266

**Pull request closed without merging**